### PR TITLE
feature/go-admin-audit-api

### LINKED ISSUE #5373: API - PKI not logging to admin audit log
**Describe the bug**
Same as title.

### PR DESCRIPTION
**Pull request by jrouzierinverse**
# Description
PKI actions are now logged to the admin api audit log

# Impacts
PKI

# Issue
fixes #5373

# Delete branch after merge
YES

# Checklist
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries
## Enhancements
* PKI actions are now logged to the admin api audit log

### CODE REVIEW

File: go/admin_api_audit_log/db.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package admin_api_audit_log
+
+import (
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/mysql"
+	"time"
+)
+
+/*
```
Comment (julsemaan) [inline go/admin_api_audit_log/db.go]: Having this here is a risk of it being unmaintained and out of sync with the actual schema so I'd say we just remove it from this file

**Comment by extrafu** [comment]
@julsemaan OK to merge?

**Comment by julsemaan** [comment]
> @julsemaan OK to merge?

My understanding was that this needed adjustments for the new API endpoints @fdurand is introducing for the renewal. @jrouzierinverse, is this going to happen in this PR or in #6844?

**Comment by jrouzierinverse** [comment]
I will do it in this PR after #6844 is merged

**Comment by extrafu** [comment]
@jrouzierinverse Done - go ahead quickly so we merge this one today.

**Comment by jrouzierinverse** [comment]
done

**Pull request merged**